jetty-server/src/main/java/org/eclipse/jetty/server/LowResourceMonitor.java

```diff
@@ -40,53 +42,44 @@
 
 
 /**
- * A monitor for low resources
- * <p>
- * An instance of this class will monitor all the connectors of a server (or a set of connectors
- * configured with {@link #setMonitoredConnectors(Collection)}) for a low resources state.
- * <p>
- * Low resources can be detected by:
+ *
+ * A monitor for low resources, low resources can be detected by:
  * <ul>
  * <li>{@link ThreadPool#isLowOnThreads()} if {@link Connector#getExecutor()} is
  * an instance of {@link ThreadPool} and {@link #setMonitorThreads(boolean)} is true.</li>
  * <li>If {@link #setMaxMemory(long)} is non zero then low resources is detected if the JVMs
  * {@link Runtime} instance has {@link Runtime#totalMemory()} minus {@link Runtime#freeMemory()}
  * greater than {@link #getMaxMemory()}</li>
- * <li>If {@link #setMaxConnections(int)} is non zero then low resources is dected if the total number
+ * <li>If {@link #setMaxConnections(int)} is non zero then low resources is detected if the total number
  * of connections exceeds {@link #getMaxConnections()}.  This feature is deprecated and replaced by
  * {@link ConnectionLimit}</li>
  * </ul>
- * <p>
- * Once low resources state is detected, the cause is logged and all existing connections returned
- * by {@link Connector#getConnectedEndPoints()} have {@link EndPoint#setIdleTimeout(long)} set
- * to {@link #getLowResourcesIdleTimeout()}.  New connections are not affected, however if the low
- * resources state persists for more than {@link #getMaxLowResourcesTime()}, then the
- * {@link #getLowResourcesIdleTimeout()} to all connections again.  Once the low resources state is
- * cleared, the idle timeout is reset to the connector default given by {@link Connector#getIdleTimeout()}.
- * <p>
- * If {@link #setAcceptingInLowResources(boolean)} is set to false (Default is true), then no new connections 
- * are accepted when in low resources state.
+ *
  */
 @ManagedObject ("Monitor for low resource conditions and activate a low resource mode if detected")
 public class LowResourceMonitor extends AbstractLifeCycle
```

Comment:
It should now extend ContainerLifeCycle and the checkers should be added as managed beans (see the update utility methods)